datamol/io.py

```diff
@@ -274,6 +274,52 @@ def to_sdf(
                 writer.write(mol)
             writer.close()
 
+def read_mol2file(
+    urlpath: Union[str, os.PathLike, IO],
+    sanitize: bool = True,
+    cleanupSubstructures: bool = True,
```

Comment:
@Pakman450 can you use snake case (`_` ) for consistency ?

Reply:
Oh Snap. You got it. Ill change it now.

Reply:
I changed to snake case. Thanks for the feedback

Reply:
I think @maclandrol was referring to `cleanupSubstructures` -> `cleanup_substructures`. It's ok to keep `read_mol2file` IMO as it follows the same pattern as the other functions such as `read_pdbfile`.

Reply:
I see. I will make changes.